src/content/reference/react/useImperativeHandle.md

```diff
@@ -105,7 +104,7 @@ const MyInput = forwardRef(function MyInput(props, ref) {
 });
 ```
 
-Now, if the parent component gets a ref to `MyInput`, it will be able to call the `focus` and `scrollIntoView` methods on it. However, it will not have full access to the underlying `<input>` DOM node.
+এখন, যদি প্যারেন্ট কম্পোনেন্ট `MyInput` এ একটি ref নিয়ে যায়, এটা `focus` এবং `scrollIntoView` মেথডগুলোকে এর উপর কল করতে পারবে। যদিও, এটা পর্দার পেছনের `<input>` DOM নোডের সম্পূর্ণ access পাবে না।
```

Comment:
এখন, যদি প্যারেন্ট কম্পোনেন্ট `MyInput` এ একটি ref নিয়ে যায় => এখন, যদি প্যারেন্ট কম্পোনেন্ট `MyInput` এ একটি ref পেয়ে যায়

Reply:
Thanks for the in-depth checking. Change requests resolved.